src/doc/extend.md

```diff
@@ -25,9 +25,7 @@ not everything fits with everyone's needs.
 ### Smart App Banners in iOS 6+ Safari
 
 Stop bothering everyone with gross modals advertising your entry in the
-App Store. Include the following [meta tag](https://developer.apple.com/library/IOS/documentation/AppleApplications/Reference/SafariWebContent/PromotingAppswithAppBanners/PromotingAppswithAppBanners.html#//apple_ref/doc/uid/TP40002051-CH6-SW2)
-will unintrusively allow the user the option to download your iOS app,
-or open it with some data about the user's current state on the website.
+App Store. Including the following [meta tag](https://developer.apple.com/library/content/documentation/AppleApplications/Reference/SafariWebContent/PromotingAppswithAppBanners/PromotingAppswithAppBanners.html) will unintrusively give the user the option to download your iOS app,or open it with some data about the user's current state on the website.
```

Comment:
"unintrusively" > "unobtrusively"

Reply:
You are right.

Reply:
Do I need to close this PR and open a new PR for this?

Reply:
Sorry, I was away for the weekend. @byog you can just push new code to your branch, You can keep the same PR.

Reply:
I have already modified it on my branch. What next?

Reply:
@byog That's all it takes. Changes to a PR branch are automatically reflected in the PR.